windows/activities.md

```diff
@@ -81,6 +81,8 @@ List of the things that B.E.N.J.I. can do:
 * Podcast
 * Calculator
 * Twitter(get top 10 tweets, get list of people you are following)
+* Say "spanish", and then give the commands in Spanish
```

Comment:
Capital `Spanish` and `French`

Reply:
@hmangukia You missed this capitalization as well :)

Reply:
Yes, updated :)